### PR TITLE
add detail version support for APPROX_PERCENTILE function in aggregate-group-by-functions.md

### PR DESCRIPTION
**Pull request by yiduoyunQ**
add detail version support for APPROX_PERCENTILE function in aggregate-group-by-functions.md

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
added

customer use tidb:v4.0.1 and try to use APPROX_PERCENTILE function but got error: FUNCTION does not exist.

### Which TiDB version(s) do your changes apply to? (Required)
release-4.0

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@yiduoyunQ Please sign the [Contributor License Agreement](https://cla-assistant.io/pingcap/docs-cn?pullRequest=6878) for this PR to be merged. Thanks!

**Comment by yiduoyunQ** [comment]
@TomShawn  done

**Review by TomShawn — APPROVED** [review]
LGTM

File: functions-and-operators/aggregate-group-by-functions.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ TiDB 支持的 MySQL `GROUP BY` 聚合函数如下所示：
 
     该函数中，表达式的返回结果必须为[数值类型](/data-type-numeric.md)或[日期与时间类型](/data-type-date-and-time.md)。函数不支持计算其他类型的返回结果，并直接返回 `NULL`。
 
+    该函数在 v4.0.8 版本开始支持使用。
```
Comment (TomShawn) [inline functions-and-operators/aggregate-group-by-functions.md:42]:
```suggestion
    该函数从 v4.0.8 版本开始支持使用。
```

**Pull request merged**